libreoffice-core/desktop/source/lib/init.cxx

```diff
@@ -8,8 +8,11 @@
  */
 
 #include "comphelper/seqstream.hxx"
+#include "editeng/editids.hrc"
+#include "editeng/formatbreakitem.hxx"
 #include "lib/unov8.hxx"
 #include <sal/backtrace.hxx>
+#include "sfx2/QuerySaveDocument.hxx"
```

Comment:
I'm guessing you didn't add these intentionally. Are these required?

Reply:
Sorry yes removed the ones that where not necessary.

Reply:
I think clangd added these automatically while I was playing around with some stuff.